x-pack/plugins/alerting/server/usage/lib/get_telemetry_from_kibana.ts

```diff
@@ -490,3 +505,57 @@ export async function getTotalCountInUse({
     };
   }
 }
+
+export async function getMWTelemetry({
+  savedObjectsClient,
+  logger,
+}: MWOpts): Promise<GetMWTelemetryResults> {
+  try {
+    const MWFinder = savedObjectsClient.createPointInTimeFinder<MaintenanceWindowAttributes>({
+      type: MAINTENANCE_WINDOW_SAVED_OBJECT_TYPE,
+      namespaces: ['*'],
+      perPage: 100,
+    });
+
+    let countMWTotal = 0;
+    let countMWWithRepeatToggleON = 0;
+    let countMWWithFilterAlertToggleON = 0;
+    for await (const response of MWFinder.find()) {
+      for (const MWSavedObject of response.saved_objects) {
+        countMWTotal = countMWTotal + 1;
+        // scopedQuery property will be null if "Filter alerts" toggle will be off
+        if (MWSavedObject.attributes.scopedQuery) {
+          countMWWithFilterAlertToggleON = countMWWithFilterAlertToggleON + 1;
+        }
+        // interval property will be not in place if "Repeat" toggle will be off
+        if ('interval' in MWSavedObject.attributes.rRule) {
```

Comment:
nit: What do you think of `Object.hasOwn(mwSavedObject.attributes.rRule, 'interval')`?

Reply:
Why do you prefer one to another?